jobtracker.go

```diff
@@ -102,7 +98,7 @@ func (jt *jobTracker) runningJobLoop() {
 	}
 
 	for range time.Tick(*pollInterval) {
-		log.Printf("Listing running jobs in cluster %s on resource manager %s\n", jt.clusterName, jt.rm)
+		log.Printf("Listing running jobs in cluster %s\n", jt.clusterName)
```

Comment:
it's kinda important to keep the piece, as the RM can hot-swap itself. do we have something jt.jobClient.getResourceManager() available?